File: bin/benchmark/prime.py
import time

t1 = time.time()

def isprime(n):
    nn = n - 1
    for i in range(2,nn):
        if n % i == 0:
            return False
    return True


def primes(n):
    count = 0

    for i in range(2,n):
        if isprime(i):
            count = count + 1
    return count


N = 10 * 10000

print(primes(N),time.time() - t1)

File: bin/benchmark/prime.cpp


var t1 = time.gettickcount()

var _isprime(var n){
    for (var i : xrange(2,n - 1))
        if (n % i == 0)
            return false
    return true
}


var primes(var n){
    var count = 0

    for (var i : xrange(2,n)){
        if (_isprime(i))
            count++;
	}
    return count
}

var N = 10 * 10000;

println(primes(N),time.gettickcount() - t1)